### DISCUSSION TITLE
Saved credit card cvc confirmation

### DISCUSSION THREAD

**Discussion by esujevic** [flutter-stripe/flutter_stripe#1517] [Q&A]
How to include cvc token when confirming payment using saved credit card? please help im losing money becuase of hackers

**Comment by linokriza** [comment]
bump

**Comment by remonh87** [comment]
You can use this method: https://github.com/flutter-stripe/flutter_stripe/blob/306f6e10b97a1a7f897e0a1b742e1abae8cec000/packages/stripe/lib/src/stripe.dart#L427 .  I have no idea how this actually works and if you experience security issues I would recommend contacting Stripe support so they can advise you. There are many things that you can do against hackers.

**Reply by esujevic** [reply]
I know for that method, but how to include that token into confirmPayment method?
